src/layouts/AuthLayout/AuthLayout.styles.js

```diff
@@ -0,0 +1,8 @@
+import styled from '@emotion/styled';
+
+export const AuthContainer = styled.main`
+  padding-top: ${({ noHeader, theme }) =>
+    noHeader ? '' : `${theme.height.header}`};
```

Comment:
isn't this declaring `padding-top` but leaving it without any value ?

 si `noHeader === true`
```padding-top: ;```

Reply:
Yes 🤦🏻‍♀️. I'm gonna change it to `null` and emotion will remove it 💪🏻  thanks!